deploy/manifests/cubeWebhook.yaml

```diff
@@ -100,3 +100,27 @@ webhooks:
         resources:
           - cuberesourcequota
     sideEffects: None
+  - admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      caBundle: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURFekNDQWZ1Z0F3SUJBZ0lKQU40VS9NcUlvNHR0TUEwR0NTcUdTSWIzRFFFQkN3VUFNQ0F4SGpBY0JnTlYKQkFNTUZTb3VhM1ZpWldOMVltVXRjM2x6ZEdWdExuTjJZekFlRncweU1UQTBNamN3TmpBNU1qRmFGdzAwT0RBNQpNVEl3TmpBNU1qRmFNQ0F4SGpBY0JnTlZCQU1NRlNvdWEzVmlaV04xWW1VdGMzbHpkR1Z0TG5OMll6Q0NBU0l3CkRRWUpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFPc2YyWEdJMmNtQkZSbXVJdTNLTUFTcCt2bWkKdWN6WlpxZ1ljV3JXUUcyNUY0aG9FU1BxRFFJRHVkTlVIMFpZWUFGbExieEllSWhnMEVhWFZmU2NuOVUxMFFEMwpqYmp6dFVBWS9mQlNsMEltaXNkWTU2QjVEYWhxdUNuNTA5Vk9OR2lSYUErL1hHWTE0djZMbElSZGJlUWlONE1JCmtMenloaVd2NVNtYTBhSTB0Q1YybkFia0QyR0Y2dU9yMHZWK2ZxVGwzR1FDWHhmUzhuZkRNWWxwQkRidFFjUTUKc3k3OXZUSzhnOWtOM3dsVEdTeENuaC9MbUtQR0lBRDNLeDdSQy9mTnhMdDJIU0tpRFN2Y1c1bzhHbGV0amoxaQpVT0MxR0tOSzRmM1FDb29EVjYycmdBOFJINDU4a2RpVlNyY0NkaWpvN2ZOMDc4YWMreExsT1BxTmc3OENBd0VBCkFhTlFNRTR3SFFZRFZSME9CQllFRkV6SkdidHhqbWs5eWRaMVIvclhkUy9BL2ZaSU1COEdBMVVkSXdRWU1CYUEKRkV6SkdidHhqbWs5eWRaMVIvclhkUy9BL2ZaSU1Bd0dBMVVkRXdRRk1BTUJBZjh3RFFZSktvWklodmNOQVFFTApCUUFEZ2dFQkFIaDJVejY4Z0YyRUlScTdPOGVyQVlQeVpqRWdCL3VjdE0ybThvYnFtelBzWHVnMXZxZk9udFVGClVONWsxZFBWY2J2djM0cHE3Y29UcnpsL0JtdnVhVTRCakJ0VzNLanJKSVJla1JmbkJxdU5ja05UMVpGWEtOUHgKUTAyU2o2MWpnMHVRazBBeG9FeFM0aUtYZ2Y1REdnck5rdWJGNGZ3S1JuajJ4SmJIWVVpUkdjRVRlQW9lNXI1dAptWCtnYjJNVTdQZktwQnVYTC9GV3hVNS9uNVY4S2xnTVMvdTlDVzhSTzhuZ24wTXlUWFdmd0FJWVpVTGRPMU9BCnBIT09zVUdqcmIrUVEwblFkL1V5aGZOSE9ueG9HRUNldFdiOU9tZGxSWWRXN1hROS9wZjVlZThqS1d3MmFESWgKZVBIdmYvZUFvQmpIS1k5dWNhUUNhNmdTM3pkQlA3VT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQ==
+      service:
+        name: kubecube
+        namespace: kubecube-system
+        port: 9443
+        path: /validate-cluster-kubecube-io-v1-cluster
+    failurePolicy: Fail
+    name: vcluster.kb.io
+    rules:
+      - apiGroups:
+          - cluster.kubecube.io
+        apiVersions:
+          - v1
+        operations:
+          - CREATE
+          - UPDATE
+          - DELETE
```

Comment:
dont define DELETE operations if we dont care at all

Reply:
fixed